src/BAP/SimpleBTSBundle/Entity/Issue.php

```diff
@@ -0,0 +1,584 @@
+     */
+    public function getType()
+    {
+        return $this->type;
+    }
+
+    /**
+     * Returns the unique taggable resource identifier
+     *
+     * @return string
+     */
+    public function getTaggableId()
+    {
+        return $this->getId();
+    }
+
+    /**
+     * Returns the collection of tags for this Taggable entity
+     *
+     * @return \Doctrine\Common\Collections\ArrayCollection
+     */
+    public function getTags()
+    {
+        return $this->tags;
+    }
+
+    /**
+     * Set tag collection
+     *
+     * @param $tags
+     * @return $this
+     */
+    public function setTags($tags)
+    {
+        $this->tags = $tags;
+
+        return $this;
+    }
+
+    /**
+     * @return User
+     */
+    public function getAssignee()
+    {
+        return $this->assignee;
+    }
+
+    /**
+     * @param User $assignee
+     * @return $this
+     */
+    public function setAssignee($assignee)
+    {
+        $this->assignee = $assignee;
+
+        return $this;
+    }
+
+    /**
+     * @return User
+     */
+    public function getReporter()
+    {
+        return $this->reporter;
+    }
+
+    /**
+     * @param User $reporter
+     * @return $this
+     */
+    public function setReporter($reporter)
+    {
+        $this->reporter = $reporter;
+
+        return $this;
+    }
+
+    /**
+     * Set createdAt
+     *
+     * @param \DateTime $createdAt
+     * @return Issue
+     */
+    public function setCreatedAt($createdAt)
+    {
+        $this->createdAt = $createdAt;
+
+        return $this;
+    }
+
+    /**
+     * Get createdAt
+     *
+     * @return \DateTime 
+     */
+    public function getCreatedAt()
+    {
+        return $this->createdAt;
+    }
+
+    /**
+     * Set updatedAt
+     *
+     * @param \DateTime $updatedAt
+     * @return Issue
+     */
+    public function setUpdatedAt($updatedAt)
+    {
+        $this->updatedAt = $updatedAt;
+
+        return $this;
+    }
+
+    /**
+     * Get updatedAt
+     *
+     * @return \DateTime 
+     */
+    public function getUpdatedAt()
+    {
+        return $this->updatedAt;
+    }
+
+    /**
+     * Add relatedIssues
+     *
+     * @param Issue $relatedIssues
+     * @return Issue
+     */
+    public function addRelatedIssue(Issue $relatedIssues)
+    {
+        $this->relatedIssues[] = $relatedIssues;
+
+        return $this;
+    }
+
+    /**
+     * Remove relatedIssues
+     *
+     * @param Issue $relatedIssues
+     */
+    public function removeRelatedIssue(Issue $relatedIssues)
+    {
+        $this->relatedIssues->removeElement($relatedIssues);
+    }
+
+    /**
+     * Get relatedIssues
+     *
+     * @return ArrayCollection|Issue[]
+     */
+    public function getRelatedIssues()
+    {
+        return $this->relatedIssues;
+    }
+
+    /**
+     * Add relatedToIssues
+     *
+     * @param Issue $relatedToIssues
+     * @return Issue
+     */
+    public function addRelatedToIssue(Issue $relatedToIssues)
```

Comment:
Why parameter has name $relatedToIssues? I think $relatedToIssue is better here because here you set only one object, not an array